music21/musicxml/test_m21ToXml.py

```diff
@@ -419,9 +419,13 @@ def testMeasurePadding(self):
         s = stream.Score([converter.parse('tinyNotation: 4/4 c4')])
         s[stream.Measure].first().paddingLeft = 2.0
         s[stream.Measure].first().paddingRight = 1.0
+        # workaround until getET() helper starts calling fromGeneralObject
+        s = GeneralObjectExporter().fromGeneralObject(s)
```

Comment:
We really need to get all the musicxml tests to declare whether or not they depend on notation being made. This was a problem _before_ we added makeNotation=False as an option -- essentially, getET() bypasses part of the normal musicxml export path.